sys/kern/main.c

```diff
@@ -110,6 +114,10 @@ __noreturn void kernel_init(void) {
 
   klog("Kernel initialized!");
 
+#if KCSAN
+  kcsan_ready = 1;
+#endif
```

Comment:
Perhaps having a `kcsan_enable()` function would look cleaner? The function would be defined as `__nothing` if `!KCSAN`.

Reply:
Or `init_kcsan()` to stay consistent with KASAN

Reply:
I like the idea. Thanks